### ISSUE TITLE
Item Lurker: Incorrect resistance value shown on some Two Stone rings

### ISSUE THREAD

**Issue by alexberryman** [C1rdec/Poe-Lurker#156]
**Describe the bug**
A few types of Two Stone rings only display the resistance for one elemental resist and ignore the other elemental resist from the implicit stat. Other Two Stone rings seem to calculate correctly so this seems to be an edge case. 

**To Reproduce**
1. Find a ring with the following stats:
```
Rarity: Rare
Brood Grasp
Two-Stone Ring
--------
Requirements:
Level: 38
--------
Item Level: 57
--------
+13% to Fire and Cold Resistances (implicit)
--------
Adds 8 to 18 Cold Damage to Attacks
+14 to maximum Energy Shield
21% increased Mana Regeneration Rate
12% increased Rarity of Items found
+31% to Fire Resistance
+4 Mana gained on Kill
```
2. Hover over the ring and activate the Item Lurker hotkey 
3. Notice that the incorrect cold resistance is displayed as 0% instead of the expected 13%, but the fire resistance is correctly using the implicit 13%+31% = 44%

**Expected behavior**
I expect Item Lurker to display 13% cold resist (instead of 0%) and 44% fire resist for this item

**Screenshots**
Incorrect cold resistance is displayed as 0% instead of the expected 13%
![no-cold-resist-listed](https://user-images.githubusercontent.com/1881846/84584796-62c27200-adce-11ea-8960-fe9468bbf782.PNG)

**Please complete the following information:**
 - Version: 1.5.30
- Provide Log (C:\Users\[user]\AppData\Local\PoeLurker\app-[version]\logs): None have been generated even after turning on debug mode

**Additional context**
I'll keep the item if you want to use it for debugging.

**Comment by C1rdec** [comment]
Thx I'm on it

**Comment by C1rdec** [comment]
Fixed in 1.6.1
